code/game/gamemodes/scoreboard.dm

```diff
@@ -20,6 +20,17 @@
 			completions += "<BR>The following explosive implants were used:<BR>"
 			for(var/entry in score["implant_phrases"])
 				completions += "[entry]<BR>"
+				
+	var/list/gallery = score["global_paintings"]
+	var/painting_completions = ""
+	if(gallery.len)
+		for(var/obj/structure/painting/custom/painting in gallery)
+			var/icon/flat = getFlatIcon(painting)
+			if(painting.show_on_scoreboard)
```

Comment:
```suggestion
			if(painting.show_on_scoreboard && !painting.blank)
```
You'll likely want to check against blank paintings here, in case a player builds a handful of canvases but never gets to paint on any of them, or a clown goes around erasing players' paintings with soap. Checking `painting.blank` will work for custom paintings both as structures and as items, or for a more future-proof option that should work on any objects that allow painting you could try getting the `/datum/custom_painting` (in this case `painting.painting_data`) and calling `is_blank()`

Reply:
yeah, noticed that, have it on my local branch, good call though :+1: